### PR TITLE
add compression for cards as well as header

### PR DESCRIPTION
**Pull request by Jesuitman**
What I did:
I added the ability to grow and shrink the information displayed I the cards by clicking them. i combined it with the changes mary had made and it allowed everything to have the same nice links of marys work while keeping a more streamlined look.

Did this break anything?

- [x] No
- [ ]  Yes

Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x]  Styling 
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [x]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?
Router, styling, Oauth, and testing!

### CODE REVIEW

File: src/App/App.css [hunk 1]
```diff
@@ -1,44 +1,25 @@
...
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.login-button {
+  background-color: grey;
+  color: white;
```
Comment (MaryBruff) [inline src/App/App.css:15]: Brilliant addng the placeholder it makes it really feel like its coming to life!

File: src/Card/Card.js [hunk 2]
```diff
@@ -1,33 +1,25 @@
...
     <div className="card">
-        {/* <img src={star} alt="Star Icon" className="star-icon"/> */}
-        <div className="card-content">
-        <p
-          dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(sanitizedHtml)}}
-        />
-      {/* You can add more information or formatting here */}
+      <div className="card-content" onClick={handleTitleClick}>
+        <p dangerouslySetInnerHTML={{ __html: snippetToShow }} />
+        <button className="saveButton">😡Save Controversy😡</button>
```
Comment (MaryBruff) [inline src/Card/Card.js:21]: Glad you were able to rope these back in there! They look great and are on brand for our product.

File: src/Card/Card.js [hunk 3]
```diff
@@ -1,33 +1,25 @@
...
+const Card = ({ snippet }) => {
+  const [showFullContent, setShowFullContent] = useState(false);
 
-const Card = ({ title, snippet }) => {
-    const [showPage,setShowPage] = useState(false)
+  const sanitizedSnippet = DOMPurify.sanitize(modifyRelativeUrls(snippet));
 
-  
-    const sanitizedHtml = DOMPurify.sanitize(modifyRelativeUrls(snippet));
+  const snippetToShow = showFullContent ? sanitizedSnippet : sanitizedSnippet.slice(0, 2500) + '...';
```
Comment (MaryBruff) [inline src/Card/Card.js:11]: Ah! So this is how you did this, not too complicated! I really like this feature and I think it adds to a flawless user experience!

**Comment by MaryBruff** [comment]
Great work Lex! It feels like its coming to life!! Stoked to get Router setup and the favoriting functionality.

File: src/Card/Card.js (hunk 3 above)
Comment (tenthwalker) [reply src/Card/Card.js:11]: I see you using that ternary operator, nice job!

**Review by tenthwalker — APPROVED** [review]
Looks solid, once we get some more styling and Oauth incorporated we'll be just about there for mvp 🎉

**Pull request merged**